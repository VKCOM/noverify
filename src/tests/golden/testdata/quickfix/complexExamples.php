<?php

function foo(array $a) { return 0; }

$_ = array(isset($x) ? $x : $y); // expressions in global scope
$_ = foo(array($x ? $x : 5));
$_ = [
    array(foo(array($x ? $x : 5))),
    array(foo([$x ? $x : 5])),
    [foo(array($x ? $x : 5))],
    [[$x ? $x : 5]],
];

function f() {
    $_ = [
        array(foo(array($x ? $x : 5))),
        array(foo([$x ? $x : 5])),
        [foo(array($x ? $x : 5))],
        [[$x ? $x : 5]],
    ];
    $_ = array(isset($x) ? $x : $y); // and in block
}
