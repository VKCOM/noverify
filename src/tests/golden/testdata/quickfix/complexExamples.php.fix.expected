<?php

function foo(array $a) { return 0; }

$_ = [isset($x) ? $x : $y]; // expressions in global scope
$_ = foo([$x ? $x : 5]);
$_ = [
    [foo(array($x ? $x : 5))],
    [foo([$x ? $x : 5])],
    [foo([$x ? $x : 5])],
    [[$x ?: 5]],
];

function f() {
    $_ = [
        [foo(array($x ? $x : 5))],
        [foo([$x ? $x : 5])],
        [foo([$x ? $x : 5])],
        [[$x ?: 5]],
    ];
    $_ = [isset($x) ? $x : $y]; // and in block
}
